llvm/tools/llvm-link/llvm-link.cpp

```diff
@@ -134,6 +134,12 @@ static cl::opt<bool> TryUseNewDbgInfoFormat(
     cl::desc("Enable debuginfo iterator positions, if they're built in"),
     cl::init(false));
 
+namespace llvm {
```

Comment:
Can we avoid this by moving the def of this flag somewhere common to all uses?

Reply:
https://github.com/llvm/llvm-project/pull/83665/commits/46ddde05cfcb8c141c88d5a7b392441f4e2f3883
Move `EnableMemProfContextDisambiguation` from PassBuilderPipelines.cpp to MemProfContextDisambiguation.cpp. Now there only one definition of `EnableMemProfContextDisambiguation`.